panther_analysis_tool/main.py

```diff
@@ -410,6 +422,11 @@ def classify_analysis(
                 SchemaUnexpectedTypeError) as err:
             invalid_specs.append((analysis_spec_filename, err))
             continue
+        except Exception as err:  # pylint: disable=broad-except
+            # Catch arbitrary exceptions thrown by bad specification files
+            logging.warn('Unexpected schema validation error "{}", please report this error message to our public repo'.format(err))
```

Comment:
```suggestion
            logging.warn('Unexpected config validation error "{}"'.format(err))
```

What's the logic around posting to our public repo? What do you mean

Reply:
My thought process is that there are some exceptions we expect from time to time (such as bad schemas), and we're ok with discarding those immediately. But there may be some other exceptions that we want to handle in different ways, we just don't know that because we've never run into them before. I can just treat those exceptions the same as the ones we are expecting to happen and ignore them, but then it just might be slightly harder to track down certain issues in the future. I can remove the message.